docs/source/release/v6.7.0/Reflectometry/Bugfixes/34812.rst

```diff
@@ -0,0 +1 @@
+- Changed the error message when the Polarization Corrections for PolRef are set using the Parameter File. The error now explains the required information is missing.
```

Comment:
Although we've done this at the request of POLREF, this change will actually apply for any instrument where users try to run a correction using a parameter file without the relevant settings defined. Would it be worth just altering the wording of the release note slightly to make it clear that the change has a wider impact than just POLREF?